mps_youtube/main.py

```diff
@@ -3280,6 +3287,31 @@ def yt_url(url, print_title=0):
     if print_title:
         xprint(v.title)
 
+def yt_url_file(file_name):
+    """ Access a list of urls in a text file """
+
+    #Open and read the file
+    try:
+        fo = open(file_name, "r")
+        lines = fo.readlines()
+    except (IOError):
+        g.message = c.r + 'Error while reading the file' + c.w
+        g.content = g.content or generate_songlist_display(zeromsg=g.message)
+        return
+
+    output = ""
```

Comment:
You can also use `[line.strip('\n') for line in fo]`, instead of calling `fo.readlines()`.